utilities/bulk-delete-projects/index.py

```diff
@@ -0,0 +1,109 @@
+import csv
+import requests
+from datetime import datetime
+import sys
+import time
+import json
+
+# ------------------------------------------------------------
+# Configuration ⚙️⚙️⚙️
+# ------------------------------------------------------------
+
+ORGSLUG = "yourorgsluggoeshere"  # Replace with your organization slug (found in Settings > Identifiers)
+BEARER_TOKEN = "yourkeygoeshere"  # Replace with your bearer token (generate one in Settings > Tokens)
+
+# ------------------------------------------------------------
+# NO EDITING BELOW THIS LINE
+# ------------------------------------------------------------
+
+API_ENDPOINT = "https://semgrep.dev/api/v1/deployments/{deployment_slug}/projects/{project_name}"
+
+def extract_error_message(response_text):
+    """Extract error message from JSON response"""
+    try:
+        error_json = json.loads(response_text)
+        if "error" in error_json:
+            return error_json["error"]
+        return response_text
+    except json.JSONDecodeError:
+        return response_text
+
+def delete_project(project_name):
+    """
+    Delete a project using the Semgrep API
+    Returns a tuple of (success, message)
+    """
+    url = API_ENDPOINT.format(
+        deployment_slug=ORGSLUG,
+        project_name=project_name
+    )
+    
+    headers = {
+        "Authorization": f"Bearer {BEARER_TOKEN}"
+    }
+    
+    try:
+        response = requests.delete(url, headers=headers)
+        if response.status_code == 200:
+            print(f"Successfully deleted project: {project_name}")
+            return True, "deleted"
```

Comment:
I'm wondering if it's really useful to produce both an output file and printed logs. I'd generally expect one or the other, or something like a file and then a summary output like "Successfully deleted X projects, failed to delete Y projects".